processing/src/main/java/org/apache/carbondata/processing/util/CarbonLoaderUtil.java

```diff
@@ -196,8 +212,13 @@ public static void deleteStorePath(String path) {
    */
   public static void deleteLocalDataLoadFolderLocation(CarbonLoadModel loadModel,
       boolean isCompactionFlow, boolean isAltPartitionFlow) {
-    String databaseName = loadModel.getDatabaseName();
     String tableName = loadModel.getTableName();
+    deleteLocalDataLoadFolderLocation(loadModel, isCompactionFlow, isAltPartitionFlow, tableName);
```

Comment:
Define TableProcessingOperations Class, mark it as developer api, so that it is not modified.